cmd/integration/server.go

```diff
@@ -79,28 +77,3 @@ func (s *server) start(args []string) error {
 
 	return nil
 }
-
-// shutdown terminates the running node server process, and cleans up
-// all files/directories created by it.
-func (s *server) shutdown() error {
```

Comment:
I think this method should not be obsolete, just like `stop`. This feels to me like its not called where it is supposed to be called (probably in some `t.Cleanup()`s)

Reply:
Good point. Actually, `stop()` does nothing extra besides calling `shutdown()`. I will remove `stop()` and add calling `shutdown()` in `main()`.